### ISSUE TITLE
SameDiff Import Test Failures

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#5738]
Following up on this: https://github.com/deeplearning4j/deeplearning4j/issues/5718

After making a number of fixes here, https://github.com/deeplearning4j/deeplearning4j/pull/5737, I'm seeing 4 remaining failures:
**1. TFGraphTestAllLibnd4j[10]**

```
Error at [D:/jenkins/ws/dl4j-master-windows-x86_64-cpu/libnd4j/include/ops/declarable/generic/random/normal.cpp:16:0]:
RNG isn't defined for this Graph instance
java.lang.RuntimeException: Op validation failed
	at org.nd4j.nativeblas.Nd4jCpu$NativeOps.executeFlatGraphFloat(Native Method)
	at org.nd4j.nativeblas.Nd4jCpu$NativeOps.executeFlatGraphFloat(Nd4jCpu.java:1464)
	at org.nd4j.autodiff.execution.NativeGraphExecutioner.executeGraph(NativeGraphExecutioner.java:93)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.getGraphAfterExec(TFGraphTestAllHelper.java:190)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.checkOnlyOutput(TFGraphTestAllHelper.java:101)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.checkOnlyOutput(TFGraphTestAllHelper.java:93)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllLibnd4j.test(TFGraphTestAllLibnd4j.java:61)
```

**2. TFGraphTestAllLibnd4j[10]**

I have looked into this failure a bunch, added some extra tests for GatherNd and GatherV2. So far the only conclusion I have is that it's likely due to one of those two ops - or potentially a bad test.

```
java.lang.AssertionError: Predictions do not match on g_05  <Click to see difference>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.checkOnlyOutput(TFGraphTestAllHelper.java:133)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.checkOnlyOutput(TFGraphTestAllHelper.java:93)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllLibnd4j.test(TFGraphTestAllLibnd4j.java:61)
```

![image](https://user-images.githubusercontent.com/2360237/42028464-acfd6526-7b0f-11e8-89a3-0bd3073e68b0.png)

```
--- Summary ---
Variables:               9                    (9 with arrays)
Functions:               3                   
SameDiff Function Defs:  0                   

--- Variables ---
- Name -          - Array Shape -     - Output Of Function -  - Inputs To Functions -
input_1           [2, 4, 3, 2]        <none>                  [GatherV2]          
input_2           [3, 2]              <none>                  [output]            
in0               [3, 2]              <none>                  [output]            
GatherV2/indices  [2]                 <none>                  [GatherV2]          
GatherV2/axis     []                  <none>                  [GatherV2]          
GatherV2          [2, 4, 3, 2]        GatherV2(gather)        [GatherNd]          
GatherNd/indices  [3, 3]              <none>                  [GatherNd]          
GatherNd          [3, 2]              GatherNd(gather_nd)     [output]            
output            [3, 2, 3]           output(stack)                               


--- Functions ---
     - Function Name -  - Op -      - Inputs -                                  - Outputs -  
0    GatherV2           Gather      [input_1, GatherV2/indices, GatherV2/axis]  [GatherV2]   
1    GatherNd           GatherNd    [GatherV2, GatherNd/indices]                [GatherNd]   
2    output             Stack       [GatherNd, in0, input_2]                    [output]     
```

**3. TFGraphTestAllLibnd4j[23]**

```
...
Node_53 mapped to layer_23
Node name: [parallel_stack/ParallelConcat]
Unknown F operation requested by hash: [0]
Can't find operation: 0

java.lang.RuntimeException: Can't find requested operation

	at org.nd4j.nativeblas.Nd4jCpu$NativeOps.executeFlatGraphFloat(Native Method)
	at org.nd4j.nativeblas.Nd4jCpu$NativeOps.executeFlatGraphFloat(Nd4jCpu.java:1464)
	at org.nd4j.autodiff.execution.NativeGraphExecutioner.executeGraph(NativeGraphExecutioner.java:93)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.getGraphAfterExec(TFGraphTestAllHelper.java:190)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.checkOnlyOutput(TFGraphTestAllHelper.java:101)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllHelper.checkOnlyOutput(TFGraphTestAllHelper.java:93)
	at org.nd4j.imports.TFGraphs.TFGraphTestAllLibnd4j.test(TFGraphTestAllLibnd4j.java:61)
```

**4. TFGraphTestAllLibnd4j[66]**

Failure on tensor_dot_misc op:

```
java.lang.AssertionError: Predictions do not match on tensor_dot_misc expected:<[[[[    4.0000,    6.0000,    6.0000,    5.0000,    6.0000,    4.0000,    2.0000,    3.0000,    5.0000,    5.0000,    1.0000,    4.0000,    6.0000,    3.0000,    2.0000,    1.0000,    5.0000,    4.0000,    4.0000,    4.0000], 
   [    4.0000,    4.0000,    3.0000,    4.0000,    2.0000,    3.0000,    3.0000,    5.0000,    3.0000,    6.0000,    5.0000,    4.0000,    4.0000,    3.0000,    6.0000,    1.0000,    2.0000,    4.0000,    2.0000,    6.0000], 
   [    4.0000,    2.0000,    3.0000,    2.0000,    3.0000,    1.0000,    2.0000,    4.0000,    3.0000,    5.0000,    3.0000,    3.0000,    5.0000,    2.0000,    6.0000,    3.0000,    4.0000,    4.0000,    4.0000,    4.0000], 
   [    6.0000,    4.0000,    5.0000,    2.0000,    5.0000,    5.0000,    5.0000,    5.0000,    2.0000,    4.0000,    4.0000,    4.0000,    5.0000,    4.0000,    3.0000,    6.0000,    3.0000,    4.0000,    5.0000,    2.0000], 
   [    5.0000,    4.0000,    4.0000,    5.0000,    4.0000,    3.0000,    4.0000,    5.0000,    5.0000,    3.0000,    5.0000,    6.0000,    6.0000,    3.0000,    4.0000,    5.0000,    7.0000,    6.0000,    5.0000,    2.0000], 
   [    4.0000,    5.0000,    5.0000,    4.0000,    5.0000,    4.0000,    4.0000,    6.0000,    3.0000,    4.0000,    5.0000,    4.0000,    6.0000,    2.0000,    3.0000,    4.0000,    3.0000,    3.0000,    2.0000,    2.0000], 
   [    3.0000,    4.0000,    7.0000,    3.0000,    5.0000,    4.0000,    5.0000,    4.0000,    4.0000,    4.0000,    4.0000,    6.0000,    2.0000,    3.0000,    2.0000,    5.0000,    5.0000,    4.0000,    5.0000,    2.0000], 
   [    2.0000,    1.0000,    6.0000,    2.0000,    2.0000,    3.0000,    4.0000,    5.0000,    5.0000,    3.0000,    6.0000,    6.0000,    4.0000,    3.0000,    3.0000,    3.0000,    3.0000,    3.0000,    4.0000,    5.0000], 
   [    4.0000,    4.0000,    3.0000,    5.0000,    2.0000,    3.0000,    4.0000,    5.0000,    3.0000,    4.0000,    5.0000,    5.0000,    8.0000,    4.0000,    5.0000,    3.0000,    3.0000,    4.0000,    4.0000,    5.0000], 
   [    4.0000,    5.0000,    3.0000,    3.0000,    7.0000,    2.0000,    3.0000,    2.0000,    6.0000,    6.0000,    4.0000,    4.0000,    3.0000,    5.0000,    6.0000,    2.0000,    4.0000,    3.0000,    3.0000,    4.0000], 
   [    5.0000,    3.0000,    3.0000,    6.0000,    5.0000,    3.0000,    2.0000,    5.0000,    4.0000,    4.0000,    3.0000,    5.0000,    5.0000,    6.0000,    7.0000,    3.0000,    4.0000,    3.0000,    5.0000,    6.0000], 
   [    7.0000,    5.0000,    6.0000,    5.0000,    7.0000,    4.0000,    6.0000,    5.0000,    5.0000,    6.0000,    4.0000,    2.0000,    5.0000,    4.0000,    3.0000,    4.0000,    1.0000,    5.0000,    5.0000,    3.0000], 
   [    2.0000,    2.0000,    6.0000,    5.0000,    5.0000,    2.0000,    5.0000,    2.0000,    4.0000,    4.0000,    5.0000,    5.0000,    4.0000,    3.0000,    7.0000,    4.0000,    5.0000,    3.0000,    3.0000,    3.0000], 
   [    2.0000,    3.0000,    2.0000,    3.0000,    3.0000,    4.0000,    4.0000,    2.0000,    4.0000,    5.0000,    3.0000,    4.0000,    5.0000,    3.0000,    7.0000,    2.0000,    1.0000,    3.0000,    2.0000,    3.0000], 
   [    2.0000,    3.0000,    3.0000,    4.0000,    3.0000,    4.0000,    2.0000,    4.0000,    4.0000,    4.0000,    5.0000,    3.0000,    5.0000,    3.0000,    6.0000,    6.0000,    5.0000,    3.0000,    5.0000,    3.0000], 
   [    4.0000,    3.0000,    5.0000,    3.0000,    5.0000,    6.0000,    5.0000,    3.0000,    4.0000,    5.0000,    5.0000,    3.0000,    3.0000,    3.0000,    4.0000,    6.0000,    4.0000,    3.0000,    7.0000,    4.0000]], 

  [[    4.0000,    6.0000,    7.0000,    5.0000,    5.0000,    3.0000,    1.0000,    2.0000,    5.0000,    5.0000,    2.0000,    5.0000,    7.0000,    5.0000,    3.0000,    1.0000,    4.0000,    6.0000,    5.0000,    7.0000], 
   [    5.0000,    6.0000,    5.0000,    6.0000,    4.0000,    3.0000,    3.0000,    4.0000,    3.0000,    4.0000,    4.0000,    4.0000,    4.0000,    3.0000,    5.0000,    2.0000,    4.0000,    5.0000,    2.0000,    5.0000], 
   [    5.0000,    4.0000,    5.0000,    4.0000,    5.0000,    2.0000,    3.0000,    5.0000,    3.0000,    6.0000,    3.0000,    4.0000,    5.0000,    3.0000,    6.0000,    5.0000,    5.0000,    6.0000,    4.0000,    6.0000], 
   [    7.0000,    4.0000,    5.0000,    3.0000,    5.0000,    4.0000,    4.0000,    4.0000,    2.0000,    2.0000,    5.0000,    3.0000,    5.0000,    3.0000,    4.0000,    6.0000,    3.0000,    5.0000,    5.0000,    3.0000], 
   [    5.0000,    4.0000,    4.0000,    4.0000,    5.0000,    2.0000,    3.0000,    5.0000,    4.0000,    2.0000,    4.0000,    5.0000,    4.0000,    2.0000,    3.0000,    4.0000,    4.0000,    5.0000,    5.0000,    1.0000], 
   [    4.0000,    4.0000,    4.0000,    3.0000,    4.0000,    5.0000,    5.0000,    8.0000,    4.0000,    4.0000,    4.0000,    3.0000,    6.0000,    2.0000,    3.0000,    4.0000,    4.0000,    4.0000,    3.0000,    2.0000], 
   [    3.0000,    4.0000,    8.0000,    3.0000,    5.0000,    5.0000,    5.0000,    3.0000,    3.0000,    4.0000,    5.0000,    7.0000,    3.0000,    3.0000,    3.0000,    6.0000,    6.0000,    5.0000,    5.0000,    3.0000], 
   [    4.0000,    3.0000,    8.0000,    3.0000,    4.0000,    2.0000,    3.0000,    4.0000,    4.0000,    3.0000,    5.0000,    5.0000,    3.0000,    2.0000,    3.0000,    3.0000,    3.0000,    4.0000,    4.0000,    4.0000], 
   [    6.0000,    6.0000,    5.0000,    6.0000,    4.0000,    5.0000,    4.0000,    6.0000,    4.0000,    5.0000,    5.0000,    4.0000,    7.0000,    3.0000,    5.0000,    5.0000,    3.0000,    5.0000,    5.0000,    6.0000], 
   [    4.0000,    5.0000,    4.0000,    2.0000,    7.0000,    2.0000,    3.0000,    1.0000,    4.0000,    5.0000,    5.0000,    4.0000,    4.0000,    5.0000,    7.0000,    2.0000,    3.0000,    3.0000,    4.0000,    4.0000], 
   [    5.0000,    3.0000,    3.0000,    6.0000,    6.0000,    3.0000,    2.0000,    4.0000,    3.0000,    3.0000,    3.0000,    3.0000,    4.0000,    4.0000,    5.0000,    1.0000,    2.0000,    3.0000,    3.0000,    4.0000], 
   [    5.0000,    4.0000,    5.0000,    4.0000,    5.0000,    6.0000,    6.0000,    6.0000,    6.0000,    7.0000,    4.0000,    3.0000,    4.0000,    5.0000,    4.0000,    4.0000,    2.0000,    5.0000,    6.0000,    4.0000], 
   [    2.0000,    2.0000,    6.0000,    5.0000,    5.0000,    1.0000,    4.0000,    2.0000,    3.0000,    4.0000,    5.0000,    5.0000,    4.0000,    5.0000,    9.0000,    4.0000,    6.0000,    4.0000,    5.0000,    5.0000], 
   [    3.0000,    4.0000,    5.0000,    5.0000,    5.0000,    4.0000,    3.0000,    1.0000,    3.0000,    4.0000,    3.0000,    4.0000,    4.0000,    3.0000,    6.0000,    2.0000,    3.0000,    3.0000,    2.0000,    3.0000], 
   [    3.0000,    4.0000,    5.0000,    6.0000,    5.0000,    5.0000,    3.0000,    4.0000,    5.0000,    5.0000,    4.0000,    3.0000,    4.0000,    3.0000,    6.0000,    7.0000,    6.0000,    4.0000,    6.0000,    4.0000], 
   [    3.0000,    3.0000,    4.0000,    3.0000,    5.0000,    5.0000,    4.0000,    2.0000,    3.0000,    4.0000,    5.0000,    3.0000,    4.0000,    2.0000,    4.0000,    5.0000,    3.0000,    3.0000,    7.0000,    4.0000]], 

  [[    2.0000,    5.0000,    6.0000,    5.0000,    5.0000,    3.0000,    1.0000,    2.0000,    4.0000,    4.0000,    1.0000,    3.0000,    6.0000,    3.0000,    3.0000,    1.0000,    4.0000,    4.0000,    4.0000,    5.0000], 
   [    3.0000,    4.0000,    3.0000,    4.0000,    2.0000,    3.0000,    3.0000,    4.0000,    3.0000,    4.0000,    3.0000,    3.0000,    4.0000,    2.0000,    5.0000,    1.0000,    3.0000,    4.0000,    2.0000,    6.0000], 
   [    4.0000,    3.0000,    4.0000,    3.0000,    3.0000,    1.0000,    2.0000,    5.0000,    2.0000,    6.0000,    4.0000,    5.0000,    6.0000,    3.0000,    6.0000,    4.0000,    4.0000,    5.0000,    3.0000,    5.0000], 
   [    6.0000,    3.0000,    4.0000,    2.0000,    4.0000,    5.0000,    5.0000,    5.0000,    2.0000,    3.0000,    4.0000,    3.0000,    5.0000,    3.0000,    3.0000,    9.0000,    6.0000,    7.0000,    7.0000,    4.0000], 
   [    4.0000,    3.0000,    3.0000,    4.0000,    4.0000,    3.0000,    4.0000,    6.0000,    5.0000,    3.0000,    5.0000,    5.0000,    5.0000,    2.0000,    4.0000,    6.0000,    7.0000,    7.0000,    5.0000,    3.0000], 
   [    4.0000,    5.0000,    4.0000,    4.0000,    5.0000,    5.0000,    5.0000,    8.0000,    4.0000,    4.0000,    4.0000,    3.0000,    5.0000,    3.0000,    3.0000,    4.0000,    4.0000,    5.0000,    3.0000,    3.0000], 
   [    2.0000,    3.0000,    6.0000,    2.0000,    5.0000,    4.0000,    4.0000,    3.0000,    3.0000,    3.0000,    5.0000,    7.0000,    2.0000,    3.0000,    2.0000,    5.0000,    5.0000,    4.0000,    4.0000,    2.0000], 
   [    2.0000,    1.0000,    6.0000,    1.0000,    2.0000,    2.0000,    3.0000,    5.0000,    4.0000,    3.0000,    5.0000,    5.0000,    3.0000,    2.0000,    2.0000,    2.0000,    2.0000,    4.0000,    3.0000,    4.0000], 
   [    4.0000,    4.0000,    4.0000,    5.0000,    2.0000,    4.0000,    4.0000,    5.0000,    2.0000,    4.0000,    4.0000,    5.0000,    9.0000,    4.0000,    5.0000,    4.0000,    3.0000,    5.0000,    5.0000,    6.0000], 
   [    4.0000,    4.0000,    3.0000,    3.0000,    6.0000,    2.0000,    3.0000,    2.0000,    5.0000,    6.0000,    4.0000,    4.0000,    3.0000,    5.0000,    6.0000,    4.0000,    5.0000,    5.0000,    6.0000,    7.0000], 
   [    4.0000,    2.0000,    3.0000,    5.0000,    4.0000,    4.0000,    3.0000,    5.0000,    5.0000,    4.0000,    3.0000,    4.0000,    5.0000,    4.0000,    6.0000,    3.0000,    4.0000,    4.0000,    5.0000,    6.0000], 
   [    6.0000,    4.0000,    6.0000,    6.0000,    6.0000,    5.0000,    6.0000,    6.0000,    7.0000,    7.0000,    4.0000,    3.0000,    4.0000,    4.0000,    4.0000,    5.0000,    2.0000,    5.0000,    7.0000,    5.0000], 
   [    2.0000,    1.0000,    5.0000,    5.0000,    4.0000,    1.0000,    4.0000,    1.0000,    3.0000,    3.0000,    5.0000,    4.0000,    4.0000,    3.0000,    7.0000,    3.0000,    6.0000,    3.0000,    3.0000,    4.0000], 
   [    1.0000,    3.0000,    2.0000,    3.0000,    3.0000,    4.0000,    3.0000,    1.0000,    3.0000,    4.0000,    2.0000,    4.0000,    4.0000,    2.0000,    6.0000,    1.0000,    2.0000,    2.0000,    2.0000,    3.0000], 
   [    2.0000,    3.0000,    3.0000,    4.0000,    4.0000,    4.0000,    2.0000,    4.0000,    4.0000,    4.0000,    5.0000,    5.0000,    5.0000,    4.0000,    8.0000,    5.0000,    5.0000,    3.0000,    5.0000,    3.0000], 
   [    3.0000,    2.0000,    4.0000,    3.0000,    5.0000,    6.0000,    5.0000,    3.0000,    4.0000,    5.0000,    5.0000,    3.0000,    4.0000,    3.0000,    4.0000,    8.0000,    6.0000,    5.0000,    9.0000,    6.0000]]]]> but was:<[[[[    4.0000,    2.0000,    1.0000,    1.0000,    1.0000,    4.0000,   -1.0000,   -1.0000,    2.0000,    1.0000,    1.0000,         0,    1.0000,         0,   -1.0000,    1.0000,    1.0000,         0,    1.0000,    1.0000], 
   [    4.0000,    1.0000,   -1.0000,    1.0000,   -1.0000,    3.0000,         0,         0,         0,    1.0000,    5.0000,         0,   -1.0000,   -1.0000,    1.0000,    1.0000,         0,         0,   -1.0000,    2.0000], 
   [    4.0000,   -1.0000,   -1.0000,   -1.0000,         0,    1.0000,   -2.0000,         0,   -1.0000,    2.0000,    3.0000,         0,         0,   -1.0000,    2.0000,    3.0000,         0,         0,         0,    1.0000], 
   [    6.0000,         0,         0,   -1.0000,    1.0000,    5.0000,    1.0000,    1.0000,   -1.0000,         0,    4.0000,   -1.0000,         0,         0,         0,    6.0000,   -1.0000,         0,    2.0000,   -1.0000], 
   [    5.0000,         0,   -1.0000,    1.0000,    1.0000,    3.0000,         0,    1.0000,    1.0000,         0,    5.0000,    1.0000,         0,   -1.0000,         0,    5.0000,    2.0000,    1.0000,    2.0000,   -1.0000], 
   [    4.0000,         0,   -1.0000,         0,    1.0000,    4.0000,         0,    2.0000,         0,         0,    5.0000,   -1.0000,    1.0000,   -1.0000,         0,    4.0000,         0,         0,         0,   -1.0000], 
   [    3.0000,   -1.0000,    2.0000,   -1.0000,    1.0000,    4.0000,    2.0000,         0,    1.0000,    1.0000,    4.0000,    2.0000,   -3.0000,         0,   -1.0000,    5.0000,    1.0000,         0,    1.0000,   -1.0000], 
   [    2.0000,   -2.0000,    2.0000,   -1.0000,         0,    3.0000,         0,    1.0000,    1.0000,         0,    6.0000,    1.0000,   -2.0000,   -1.0000,   -1.0000,    3.0000,   -1.0000,         0,    1.0000,    1.0000], 
   [    4.0000,         0,   -1.0000,    2.0000,   -1.0000,    3.0000,         0,    1.0000,   -1.0000,   -1.0000,    5.0000,         0,    3.0000,         0,    1.0000,    3.0000,   -1.0000,         0,         0,         0], 
   [    4.0000,    1.0000,   -1.0000,   -2.0000,    1.0000,    2.0000,         0,   -2.0000,    1.0000,    1.0000,    4.0000,         0,   -1.0000,         0,    1.0000,    2.0000,         0,   -1.0000,   -1.0000,   -1.0000], 
   [    5.0000,   -1.0000,   -1.0000,    1.0000,         0,    3.0000,   -1.0000,    1.0000,         0,   -1.0000,    3.0000,         0,         0,         0,         0,    3.0000,         0,   -1.0000,         0,         0], 
   [    7.0000,         0,    1.0000,         0,         0,    4.0000,    2.0000,    1.0000,    1.0000,    1.0000,    4.0000,   -1.0000,         0,         0,   -1.0000,    4.0000,   -1.0000,    1.0000,    2.0000,   -1.0000], 
   [    2.0000,   -2.0000,    1.0000,         0,   -1.0000,    2.0000,    2.0000,   -1.0000,         0,         0,    5.0000,    1.0000,         0,   -1.0000,    2.0000,    4.0000,    2.0000,   -1.0000,   -1.0000,   -1.0000], 
   [    2.0000,         0,   -1.0000,   -1.0000,   -1.0000,    4.0000,         0,   -2.0000,         0,         0,    3.0000,         0,         0,   -2.0000,    1.0000,    2.0000,   -1.0000,   -1.0000,   -2.0000,   -1.0000], 
   [    2.0000,         0,         0,         0,   -1.0000,    4.0000,   -2.0000,         0,         0,         0,    5.0000,         0,         0,   -1.0000,    1.0000,    6.0000,    1.0000,   -1.0000,    1.0000,   -1.0000], 
   [    4.0000,   -1.0000,         0,   -1.0000,    1.0000,    6.0000,    1.0000,   -1.0000,         0,    1.0000,    5.0000,   -1.0000,   -1.0000,   -2.0000,         0,    6.0000,         0,   -1.0000,    3.0000,         0]], 

  [[    4.0000,    2.0000,    1.0000,    1.0000,    1.0000,    3.0000,   -1.0000,   -1.0000,    2.0000,    1.0000,    2.0000,         0,    1.0000,         0,   -1.0000,    1.0000,    1.0000,         0,    1.0000,    1.0000], 
   [    5.0000,    1.0000,   -1.0000,    1.0000,   -1.0000,    3.0000,         0,         0,         0,    1.0000,    4.0000,         0,   -1.0000,   -1.0000,    1.0000,    2.0000,         0,         0,   -1.0000,    2.0000], 
   [    5.0000,   -1.0000,   -1.0000,   -1.0000,         0,    2.0000,   -2.0000,         0,   -1.0000,    2.0000,    3.0000,         0,         0,   -1.0000,    2.0000,    5.0000,         0,         0,         0,    1.0000], 
   [    7.0000,         0,         0,   -1.0000,    1.0000,    4.0000,    1.0000,    1.0000,   -1.0000,         0,    5.0000,   -1.0000,         0,         0,         0,    6.0000,   -1.0000,         0,    2.0000,   -1.0000], 
   [    5.0000,         0,   -1.0000,    1.0000,    1.0000,    2.0000,         0,    1.0000,    1.0000,         0,    4.0000,    1.0000,         0,   -1.0000,         0,    4.0000,    2.0000,    1.0000,    2.0000,   -1.0000], 
   [    4.0000,         0,   -1.0000,         0,    1.0000,    5.0000,         0,    2.0000,         0,         0,    4.0000,   -1.0000,    1.0000,   -1.0000,         0,    4.0000,         0,         0,         0,   -1.0000], 
   [    3.0000,   -1.0000,    2.0000,   -1.0000,    1.0000,    5.0000,    2.0000,         0,    1.0000,    1.0000,    5.0000,    2.0000,   -3.0000,         0,   -1.0000,    6.0000,    1.0000,         0,    1.0000,   -1.0000], 
   [    4.0000,   -2.0000,    2.0000,   -1.0000,         0,    2.0000,         0,    1.0000,    1.0000,         0,    5.0000,    1.0000,   -2.0000,   -1.0000,   -1.0000,    3.0000,   -1.0000,         0,    1.0000,    1.0000], 
   [    6.0000,         0,   -1.0000,    2.0000,   -1.0000,    5.0000,         0,    1.0000,   -1.0000,   -1.0000,    5.0000,         0,    3.0000,         0,    1.0000,    5.0000,   -1.0000,         0,         0,         0], 
   [    4.0000,    1.0000,   -1.0000,   -2.0000,    1.0000,    2.0000,         0,   -2.0000,    1.0000,    1.0000,    5.0000,         0,   -1.0000,         0,    1.0000,    2.0000,         0,   -1.0000,   -1.0000,   -1.0000], 
   [    5.0000,   -1.0000,   -1.0000,    1.0000,         0,    3.0000,   -1.0000,    1.0000,         0,   -1.0000,    3.0000,         0,         0,         0,         0,    1.0000,         0,   -1.0000,         0,         0], 
   [    5.0000,         0,    1.0000,         0,         0,    6.0000,    2.0000,    1.0000,    1.0000,    1.0000,    4.0000,   -1.0000,         0,         0,   -1.0000,    4.0000,   -1.0000,    1.0000,    2.0000,   -1.0000], 
   [    2.0000,   -2.0000,    1.0000,         0,   -1.0000,    1.0000,    2.0000,   -1.0000,         0,         0,    5.0000,    1.0000,         0,   -1.0000,    2.0000,    4.0000,    2.0000,   -1.0000,   -1.0000,   -1.0000], 
   [    3.0000,         0,   -1.0000,   -1.0000,   -1.0000,    4.0000,         0,   -2.0000,         0,         0,    3.0000,         0,         0,   -2.0000,    1.0000,    2.0000,   -1.0000,   -1.0000,   -2.0000,   -1.0000], 
   [    3.0000,         0,         0,         0,   -1.0000,    5.0000,   -2.0000,         0,         0,         0,    4.0000,         0,         0,   -1.0000,    1.0000,    7.0000,    1.0000,   -1.0000,    1.0000,   -1.0000], 
   [    3.0000,   -1.0000,         0,   -1.0000,    1.0000,    5.0000,    1.0000,   -1.0000,         0,    1.0000,    5.0000,   -1.0000,   -1.0000,   -2.0000,         0,    5.0000,         0,   -1.0000,    3.0000,         0]], 

  [[    2.0000,    2.0000,    1.0000,    1.0000,    1.0000,    3.0000,   -1.0000,   -1.0000,    2.0000,    1.0000,    1.0000,         0,    1.0000,         0,   -1.0000,    1.0000,    1.0000,         0,    1.0000,    1.0000], 
   [    3.0000,    1.0000,   -1.0000,    1.0000,   -1.0000,    3.0000,         0,         0,         0,    1.0000,    3.0000,         0,   -1.0000,   -1.0000,    1.0000,    1.0000,         0,         0,   -1.0000,    2.0000], 
   [    4.0000,   -1.0000,   -1.0000,   -1.0000,         0,    1.0000,   -2.0000,         0,   -1.0000,    2.0000,    4.0000,         0,         0,   -1.0000,    2.0000,    4.0000,         0,         0,         0,    1.0000], 
   [    6.0000,         0,         0,   -1.0000,    1.0000,    5.0000,    1.0000,    1.0000,   -1.0000,         0,    4.0000,   -1.0000,         0,         0,         0,    9.0000,   -1.0000,         0,    2.0000,   -1.0000], 
   [    4.0000,         0,   -1.0000,    1.0000,    1.0000,    3.0000,         0,    1.0000,    1.0000,         0,    5.0000,    1.0000,         0,   -1.0000,         0,    6.0000,    2.0000,    1.0000,    2.0000,   -1.0000], 
   [    4.0000,         0,   -1.0000,         0,    1.0000,    5.0000,         0,    2.0000,         0,         0,    4.0000,   -1.0000,    1.0000,   -1.0000,         0,    4.0000,         0,         0,         0,   -1.0000], 
   [    2.0000,   -1.0000,    2.0000,   -1.0000,    1.0000,    4.0000,    2.0000,         0,    1.0000,    1.0000,    5.0000,    2.0000,   -3.0000,         0,   -1.0000,    5.0000,    1.0000,         0,    1.0000,   -1.0000], 
   [    2.0000,   -2.0000,    2.0000,   -1.0000,         0,    2.0000,         0,    1.0000,    1.0000,         0,    5.0000,    1.0000,   -2.0000,   -1.0000,   -1.0000,    2.0000,   -1.0000,         0,    1.0000,    1.0000], 
   [    4.0000,         0,   -1.0000,    2.0000,   -1.0000,    4.0000,         0,    1.0000,   -1.0000,   -1.0000,    4.0000,         0,    3.0000,         0,    1.0000,    4.0000,   -1.0000,         0,         0,         0], 
   [    4.0000,    1.0000,   -1.0000,   -2.0000,    1.0000,    2.0000,         0,   -2.0000,    1.0000,    1.0000,    4.0000,         0,   -1.0000,         0,    1.0000,    4.0000,         0,   -1.0000,   -1.0000,   -1.0000], 
   [    4.0000,   -1.0000,   -1.0000,    1.0000,         0,    4.0000,   -1.0000,    1.0000,         0,   -1.0000,    3.0000,         0,         0,         0,         0,    3.0000,         0,   -1.0000,         0,         0], 
   [    6.0000,         0,    1.0000,         0,         0,    5.0000,    2.0000,    1.0000,    1.0000,    1.0000,    4.0000,   -1.0000,         0,         0,   -1.0000,    5.0000,   -1.0000,    1.0000,    2.0000,   -1.0000], 
   [    2.0000,   -2.0000,    1.0000,         0,   -1.0000,    1.0000,    2.0000,   -1.0000,         0,         0,    5.0000,    1.0000,         0,   -1.0000,    2.0000,    3.0000,    2.0000,   -1.0000,   -1.0000,   -1.0000], 
   [    1.0000,         0,   -1.0000,   -1.0000,   -1.0000,    4.0000,         0,   -2.0000,         0,         0,    2.0000,         0,         0,   -2.0000,    1.0000,    1.0000,   -1.0000,   -1.0000,   -2.0000,   -1.0000], 
   [    2.0000,         0,         0,         0,   -1.0000,    4.0000,   -2.0000,         0,         0,         0,    5.0000,         0,         0,   -1.0000,    1.0000,    5.0000,    1.0000,   -1.0000,    1.0000,   -1.0000], 
   [    3.0000,   -1.0000,         0,   -1.0000,    1.0000,    6.0000,    1.0000,   -1.0000,         0,    1.0000,    5.0000,   -1.0000,   -1.0000,   -2.0000,         0,    8.0000,         0,   -1.0000,    3.0000,         0]]]]>
```

**Comment by raver119** [comment]
It must be caused by stale branch. All these issues where fixed yesterday via #5729 

I’ll double check, sure.

**Comment by raver119** [comment]
Yes, not the case since yesterday.
